vignettes/get-started.Rmd

```diff
@@ -0,0 +1,139 @@
+---
+title: Get Started with 'altdoc'
+---
+
+## Quick start
+
+To create, render, and preview a new documentation website, execute these commands from the root directory of your package:
```

Comment:
ditto about wrapping text

Reply:
You want me to wrap everything? Curious why you like text wrapping... ;)